tracee-ebpf/tracee/tracee.bpf.c

```diff
@@ -1462,6 +1462,11 @@ static __always_inline int save_path_to_str_buf(buf_t *string_p, const struct pa
     unsigned int off;
     int sz;
 
+    // Get per-cpu string buffer
```

Comment:
BPF programs can't be preempted and moved to a different CPU, right?

Reply:
Right